prow/cmd/deck/static/pr-script.js

```diff
@@ -786,6 +838,30 @@ function createMergeStatus(prLabels, queries) {
     return statusContainer;
 }
 
+/**
+ * Creates the merge conflict status.
+ * @param prMergeability
+ * @return {Element}
+ */
+function createMergeConflictStatus(prMergeability) {
+    const statusContainer = document.createElement("DIV");
+    statusContainer.classList.add("status-container");
+    const status = document.createElement("DIV");
+    if (prMergeability === "CONFLICTING") {
+        status.appendChild(createIcon("error", "", ["status-icon", "failed"]));
+        status.appendChild(
+            document.createTextNode("Has merge conflicts"));
+    } else {
+        status.appendChild(
+            createIcon("check_circle", "", ["status-icon", "succeeded"]));
+        status.appendChild(
+            document.createTextNode("Does not have merge conflicts"));
```

Comment:
possibly "Does not appear to have merge conflicts" ?  GitHub's state here is lazy IIRC